interpreter/src/functions.rs

```diff
@@ -224,6 +224,16 @@ pub fn starts_with(This(this): This<Arc<String>>, prefix: Arc<String>) -> bool {
     this.starts_with(prefix.as_str())
 }
 
+/// Returns true if a string starts with another string.
+///
+/// # Example
+/// ```cel
+/// "abc".endsWith("c") == true
+/// ```
+pub fn ends_with(This(this): This<Arc<String>>, prefix: Arc<String>) -> bool {
+    this.ends_with(prefix.as_str())
+}
```

Comment:
Probably a copy-paste, but I think this would be more accurately named `suffix`. 
```suggestion
pub fn ends_with(This(this): This<Arc<String>>, suffix: Arc<String>) -> bool {
    this.ends_with(suffix.as_str())
}
```